openapi/nexus.json

```diff
@@ -9630,7 +9631,11 @@
             "type": "string"
           },
           "memory": {
-            "$ref": "#/components/schemas/ByteCount"
+            "description": "The amount of memory to allocate to the instance, in bytes.\n\nMust be between 1 and 256 GiB.",
+            "type": "integer",
+            "format": "uint64",
+            "minimum": 1073741824,
+            "maximum": 274877906944
```

Comment:
This worked out nicely! The numbers are a bit hard to read but I think the calculated description is helpful in that regard.